languages.yaml

```diff
@@ -826,6 +826,41 @@ GdScript:
       - "#458DC0"
       - "#FFFFFF"
     chip: "#355570"
+Glsl:
+  type: programming
+  ascii: |
+    {0}                    oooo.
+    {0}          coooooooooooooooo
+    {0}        coooooccccccccccccooooo
+    {0}       oooocoooccooooooooooooooooo
+    {0}     ooooooooooooo          CooooDD
+    {0}   coooooooooo                  oooo
+    {0}   ooOooooo.{1}__     _      _____    _
+    {0} .oOooo  {1}/:::::\  |:|    /::::\\  |:|
+    {0} .ooooo {1}/::::..:| |:|   /:::**||  |:|
+    {0} oooooS {1}|::|  |*| |:|   |::|  */  |:|
+    {0} ooooo  {1}|::|  /// |:|   |:::\ //  |:|
+    {0} ooooo  {1}|::|      |:|    \:::\    |:|
+    {0} ooooo  {1}|::| ///* |:|      \:::\  |:|
+    {0} ooooo  {1}|::|  |:| |:|    // \:::\ |:|
+    {0} oooooS {1}\::\  |:| |:|_  /*  |:::| |:|_
+    {0} ooooo   {1}\::.:::| |:::| ||**/::/  |:::|
+    {0} .oSoooSS {1}\::::/  |:::|  \::::/   |:::|
+    {0}   oocooooo..                       .
+    {0}   coooooooooo                  oooo
+    {0}     ooooooo_oooo.          ooooood
+    {0}       oooocoooocooooooooooooooood
+    {0}         Soooooccccccccccccooooo
+    {0}           Soooooooooooooood
+    {0}                 ooood
```

Comment:
I took some time to clean your design a bit, please tell me what you think @sangsatori 

![image](https://user-images.githubusercontent.com/13710835/218887052-3a189539-6375-4025-9a78-0c3ced3c8689.png)

```
    {0}            aaaaaaaaaaaaaaaa
    {0}         aaaaaaaaaaaaaaaaaaaaaa
    {0}      aaaaaaaaaaaaaaaaaaaaaaaaaaa
    {0}    aaaaaaaa                aaaaaaa
    {0}  aaaaaaaa                     aaaaa
    {0} aaaaaaa                           aa
    {0}aaaaaaa {1}_____  _        _____  _     {0}a
    {0}aaaaaa {1}/:::::||:|      /:::::||:|
    {0}aaaaa {1}|:|  __ |:|     |:(___  |:|
    {0}aaaaa {1}|:| |::||:|      \::::\ |:|
    {0}aaaaa {1}|:|__|:||:|____  ____):||:|____
    {0}aaaaaa {1}\:::::||::::::||:::::/ |::::::|
    {0}aaaaaaa                              a
    {0} aaaaaaa                           aa
    {0}  aaaaaaaa                     aaaaa
    {0}    aaaaaaaa                aaaaaaa
    {0}      aaaaaaaaaaaaaaaaaaaaaaaaaaa
    {0}         aaaaaaaaaaaaaaaaaaaaaa
    {0}            aaaaaaaaaaaaaaaa
```

Reply:
Looks great, thanks for taking the time! :heart: Much cleaner and more compact. :+1:

---

IIRC noisy ASCII was from me trying to smooth the curve appearance somehow - but I never got it looking right.

I experimented with some adjustments to your suggestion (while maintaining the symmetry).
Honestly not sure if this is better, or worse. :sweat_smile: 

I trust your judgment on which one fits the project better.

![glsl-suggestion-adjusted01](https://user-images.githubusercontent.com/23723593/219096026-bde7662f-e9d1-4f77-b5c0-37fe684fe61e.png)


```suggestion
    {0}          ..@@@@@@@@@@@@@@@@.
    {0}       ..@@@@@@@@@@@@@@@@@@@@@@.
    {0}     .@@@@@@@@@@@@@@@@@@@@@@@@@@@.
    {0}   .@@@@@@@@~              ~@@@@@@@.
    {0} .@@@@@@@@                     ~@@@@.
    {0}.@@@@@@@~                         ~@@.
    {0}@@@@@@@  {1}_____  _        _____  _   {0}~@
    {0}@@@@@@  {1}/:::::||:|      /:::::||:|
    {0}@@@@@@ {1}|:|  __ |:|     |:(___  |:|
    {0}@@@@@@ {1}|:| |::||:|      \::::\ |:|
    {0}@@@@@@ {1}|:|__|:||:|____  ____):||:|____
    {0}@@@@@@  {1}\:::::||::::::||:::::/ |::::::|
    {0}@@@@@@@                             ~@
    {0}.@@@@@@@~                         ~@@.
    {0} .@@@@@@@@                     ~@@@@.
    {0}   .@@@@@@@@~              ~@@@@@@@.
    {0}     .@@@@@@@@@@@@@@@@@@@@@@@@@@@.
    {0}       ..@@@@@@@@@@@@@@@@@@@@@@.
    {0}          ..@@@@@@@@@@@@@@@@.
```

Reply:
Personally, I think that looks great :+1:

Have you pushed up a commit with this change? Not seeing it in the PR yet.

Reply:
Just did :ok_hand: https://github.com/o2sh/onefetch/pull/824/commits/cdd8bf2da78149b75e604bcdea4316fb157514fd